internal/api/fetcher/configapi_test.go

```diff
@@ -16,26 +16,60 @@ import (
 
 	"github.com/edgelesssys/constellation/v2/internal/api/configapi"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
+var testCfg = configapi.AzureSEVSNPVersion{
+	Microcode:  93,
+	TEE:        0,
+	SNP:        6,
+	Bootloader: 2,
+}
+
 func TestGetVersion(t *testing.T) {
```

Comment:
Our usual test pattern is `TestFoo` where `Foo` is the function that is being tested.
In this case that would be `FetchLatestAzureSEVSNPVersion`.